app/javascript/app.js

```diff
@@ -46,6 +46,24 @@ export default template({
         }
       })
     }),
+    this.$on('find-matching-statements', (data, cb) => {
+      const {resourceType, statement, nameAttr, itemAttr, newItem} = data
+      if (resourceType === 'person')  {
+        cb(0, 0);
+        return;
+      }
+      let otherMatching = this.statements.filter(s => {
+        return nameAttr && itemAttr && statement[nameAttr] &&
+          (s[nameAttr] === statement[nameAttr]) &&
+          (s[itemAttr] !== newItem) &&
+          (s.transaction_id != statement.transaction_id)
+      })
+      let otherMatchingUnreconciled = otherMatching.filter(s => !s[itemAttr])
+      cb({
+        otherMatching: otherMatching.length,
+        otherMatchingUnreconciled: otherMatchingUnreconciled.length
+      })
+    }),
```

Comment:
Tiny thing: I think this `,` (and the one just before this diff starts, which was already there!) aren't needed as this isn't an object. I'm surprised that nothing has complained about it, but I guess the compiler is ignoring it or something (_waves hands_)?